### ISSUE TITLE
MQQT messages are missing sensor info

### ISSUE THREAD

**Issue by jonbartlett** [GreyEarl/qingping-air-monitor-mqtt#2]
Thanks for the instructions here. I had this working fine with Home Assistant and MQQT for a few months. Recently the integration for the 2 devices I have has stopped working. Digging deeping I see the messages are coming through from the devices but in the content there is no sensor info. Here is an example of what I am seeing now:

`{
	"id":	13,
	"type":	"13",
	"wifi_info":	"windyhill-2GHz,-45,1,7E:45:58:7F:7A:41",
	"sw_version":	"4.8.1",
	"module_version":	"4.8.1",
	"mcu_version":	"4.6.2",
	"wifi_mac":	"04CF8C28AF98",
	"hk_salt":	"C316B322F432F125CD431083C5A0A63B",
	"timestamp":	368,
	"timezone":	110
}`

I have blocked any internet access from the devices so theres no chance this has been caused by an update. Just wondering if anyone else has seen the same issue?

I'll try and factory reset now.

**Comment by jonbartlett** [comment]
Factory reset and re-config via https://developer.qingping.co/ has fixed the issue. I wonder whether the Private Access Config expires?

**Comment by GreyEarl** [comment]
You went full metal jacket with a factory reset :) It should have been sufficient by just reboot (off/on). I have noticed this once in a while. Espeicially during a general power failure or a reboot of my WLAN router.

If a reboot did not work, I granted the device access to the internet for a brief moment. Though declining some specific DNS requests, Using Adguard Home.

I have three devices. I noticed this issue occured on newer firmwares and not on old firmwares versions. I have written Qingping about it a while back as feedback.
